R/parameters.R

```diff
@@ -19,18 +19,41 @@ get_parameters <- function(iso3c = NULL,
     contact_matrix_set <- squire::get_mixing_matrix(iso3c = iso3c)
   }
 
-  c(
-    squire::parameters_explicit_SEEIR(
+  parameters <- list(
+    sq = squire::parameters_explicit_SEEIR(
       population = population,
       country = get_country(iso3c),
       contact_matrix_set = contact_matrix_set,
-      dt = 1, # dt should always be 1 as individual is always discrete time
+      dt = 1, # dt should always be 1 as individual is always a discrete time
       time_period = time_period,
       ...
     ),
     time_period = time_period,
     max_age = max_age
   )
+
+  parstemp <- parameters$sq
+
+  if (is.null(parameters$max_age)) {
+    newlist <- list(time_period = parameters$time_period, max_age = 100)
+    parstemp <- append(parstemp, newlist)
+
+  }
+  else {
+    newlist <- list(time_period = parameters$time_period,
+                     max_age = parameters$max_age)
+    parstemp <- append(parstemp, newlist)
+  }
```

Comment:
we have a default max_age in the function arguments and time_period is added to `parameters` in the lines above. What is this for?